### ISSUE TITLE
On importing a file any existing time range selection is retained but transferred to the imported file

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#3685]
### Bug description

On importing a file any existing time range selection is retained but transferred to the imported file

_And this is definitely not a regression,  this has been the case in all versions of Audacity from 1.0.0 onwards_

### Steps to reproduce

1. generate a thirty second chirp
2. Select from 10 to 20 seconds
3. Import a 3 minute audio file
4. Observe: the tome range of 10-20 secs remains but is transferred to the imported file.

### Selection to remain wher the user put it at step 2
![image](https://user-images.githubusercontent.com/50205984/192136011-aaa65f6b-5b90-4bac-b5e0-9bca2cfa9fde.png)

### Selection time range retained, but transferred to the imported file
![image](https://user-images.githubusercontent.com/50205984/192135916-d629e911-e4cb-44a5-8bbc-b645c36f36f6.png)

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

### Additional context

See also:  Importing labels and audio resets zoom and scroll position #3678

**Comment by Ryan-Miller-Audacity** [comment]
Yeah I see the same behavior testing this as well. Selection within a track would be best to be retained when importing audio rather than it being overridden and selected in the imported track.
